renderer/pages/progress.js

```diff
@@ -0,0 +1,86 @@
+import { ipcRenderer } from "electron";
+import { fileSizeReadable } from "../utils";
+import Layout from "../components/Layout";
+import Logo from "../components/Logo";
+
+class Progress extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: 0,
+      transferred: 0,
+      percent: 0
+    };
+  }
+
+  componentDidMount() {
+    ipcRenderer.on("progress", (event, progressObject) => {
+      this.setState(progressObject);
+    });
+  }
+
+  render() {
+    const { total, delta, transferred, percent, bytesPerSecond } = this.state;
```

Comment:
`bytesPerSecond`, `total` and `delta` are not in your initial state. Also, you're not using them anywhere.